examples/js/archive.js

```diff
@@ -178,16 +189,100 @@ function getImageName(imageURL) {
   return imageName;
 }
 
+function extractJsonFromUrlParams(url) { 
+  const startIndex = url.lastIndexOf('=');
+  const jsonDownloadURL = url.slice(startIndex + 1);
+
+  return jsonDownloadURL;
+}
+
+function isJsonDetected(url) {
+  if (url.includes('?json=')) {
+    const startIndex = url.lastIndexOf('.');
+    const fileExtension = url.slice(startIndex + 1);
+
+    if (fileExtension === 'json') {
+      console.log('JSON found in map shareable link'); // left here purposely
+      return true;
+    } 
+  }
+
+  return false;
+}
+
+function createMap (imageURL, options, newImage = false) {
```

Comment:
Could this be called createImage? Or placeImage?

Reply:
And could it be within the core library? We aren't doing anything custom here, right? Or do you need it for other purposes in archive.js?

Reply:
> And could it be within the core library? We aren't doing anything custom here, right? Or do you need it for other purposes in archive.js?

It shouldn't be in the core library because what it does is instantiate `distortableImageOverlay`. This is a user application responsibility. As for renaming, I will do that.

Reply:
> > And could it be within the core library? We aren't doing anything custom here, right? Or do you need it for other purposes in archive.js?
> 
> It shouldn't be in the core library because what it does is instantiate `distortableImageOverlay`. This is a user application responsibility or what do you think?. As for renaming, I will do that.